<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWClass</class>
 <widget class="QMainWindow" name="MainWClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>226</width>
      <height>128</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Display cell type</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="dispCellType"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Granule start page</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="grDispStartNum"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Display single cell #</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="dispSingleCellNum"/>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="tempMetricBinN"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Temp Metric bin #</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>131</width>
      <height>197</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="dispAllCells">
       <property name="text">
        <string>display all cells</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="dispSingleCell">
       <property name="text">
        <string>display single cell</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="calcTempMetrics">
       <property name="text">
        <string>Calc temp metrics</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loadPSH">
       <property name="text">
        <string>load PSH file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loadSim">
       <property name="text">
        <string>load sim state file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="quitButton">
       <property name="text">
        <string>quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="exportSim">
       <property name="text">
        <string>export sim state</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dispAllCells</sender>
   <signal>clicked()</signal>
   <receiver>MainWClass</receiver>
   <slot>dispAllCells()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dispSingleCell</sender>
   <signal>clicked()</signal>
   <receiver>MainWClass</receiver>
   <slot>dispSingleCell()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadPSH</sender>
   <signal>clicked()</signal>
   <receiver>MainWClass</receiver>
   <slot>loadPSHFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>calcTempMetrics</sender>
   <signal>clicked()</signal>
   <receiver>MainWClass</receiver>
   <slot>calcTempMetrics()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>121</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadSim</sender>
   <signal>clicked()</signal>
   <receiver>MainWClass</receiver>
   <slot>loadSimFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exportSim</sender>
   <signal>clicked()</signal>
   <receiver>MainWClass</receiver>
   <slot>exportSim()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tempMetricBinN</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWClass</receiver>
   <slot>changeTempMetricBinN(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>dispAllCells()</slot>
  <slot>dispSingleCell()</slot>
  <slot>loadPSHFile()</slot>
  <slot>calcTempMetrics()</slot>
  <slot>loadSimFile()</slot>
  <slot>exportSim()</slot>
  <slot>changeTempMetricBinN(int)</slot>
 </slots>
</ui>
